docs/modules/nifi/pages/usage_guide/monitoring.adoc

```diff
@@ -1,4 +1,8 @@
 = Monitoring
 
-The managed NiFi cluster is automatically configured to export Prometheus metrics. See
-xref:home:operators:monitoring.adoc[] for more details.
+The operator automatically configures NiFi to export Prometheus metrics.
+This is done by creating a https://kubernetes.io/docs/concepts/workloads/controllers/job/[Job] that connects to NiFi and configures a reporting task.
```

Comment:
On some documentation pages, an external link icon is used to mark an external link:

```suggestion
This is done by creating a https://kubernetes.io/docs/concepts/workloads/controllers/job/[Job^{external-link-icon}^] that connects to NiFi and configures a reporting task.
```

I am not sure if this should be applied to all external links.

Reply:
I've experimented with this icon in some parts, yes. but it was mostly for menu entries. I would not use it here now, but I think it's probably best to come up with a policy about it and put it in the style guide. I'm making a note of it